src/changes.rs

```diff
@@ -61,20 +31,26 @@ impl ChangeBatch {
         }
     }
 
+    //TODO: Use serde
```

Comment:
I think you decided not to use serde? It was in the message coming with your PR.

Reply:
Or it is not related to this?

Reply:
We forget to remove this. Removed